### ISSUE TITLE
Add a way to customize the available "new tab" options

### ISSUE THREAD

**Issue by Isukiipko** [betterbrowser/arcfox#75]
So here on the sidebar there are 2 options baked in (other than new tab and new folder)
![image](https://github.com/user-attachments/assets/9674adbc-8395-4121-a349-0227a2223387)
could there possibly be a way to customize the available new pages on this menu?

**Comment by LuanderFarias** [comment]
Yep, we gonna add the possibility to add your own button and remove buttons when we finish the settings/library page
